handlers/kafka_helper.go

```diff
@@ -68,6 +68,7 @@ func produceKafkaMessage(paymentID string, refundID string) error {
 		err = fmt.Errorf("error creating kafka producer: [%v]", err)
 		return err
 	}
+	defer kafkaProducer.Close()
```

Comment:
It might be worth checking the error response from `Close`:

```
defer closeKafkaProducer(kafkaProducer, paymentID)

func closeKafkaProducer(kafkaProducer *producer.Producer, paymentID string) {
	err := kafkaProducer.Close()
	if err != nil {
		log.Error(fmt.Errorf("error closing kafka producer for payment ID [%s]", paymentID))
	}
}
```

Reply:
Done 👍